### ISSUE TITLE
"Selected Model" and "Model multiplier" information associate with copilot response is not accessible with screen reader: A11y_Visual Studio Code_Github_Copilot_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#266687]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yeDAD; #A11yMAS; #A11yTCS; #Win32; #A11ySev2;#DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.3.1; #FTP; #Screen Reader; #JAWS; #Element:chatboard; #STP; #NVDA; 

## Environment Details:
Application: VS Code
Visual studio code version: 1.103.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
JAWS version: 2025.2504.89
NVDA: 2024.4.2

## Repro Steps

1. Turn on Jaws/NVDA screen reader
2. Open Visual Studio Code 
3. Ensure GitHub Copilot is installed and signed in.
4. TAB till "Select model" and press ENTER key.
5. Select a model and given some prompt in GitHub copilot chat input and press ENTER key.
6. Repeat this previews step by selecting different model.
7. TAB till Copilot response
8. Observer the issue

## Actual Experience.
When there are multiple copilot responses in the chat for different selected models, the "Selected Model" and "Model model multiplier" information appears at the bottom right corner of each response. However, this information is not accessible with a screen reader when navigating through that specific response.

## Expected Experience
The "Selected Model" and "Model model multiplier" information, which is displayed in the bottom right corner of each copilot response, should be accessible to screen readers when navigating through the copilot response.

## User Impact:
When there are multiple responses for different selected models, it can be challenging for screen reader users to identify which model is associated with each response.

## Attachment
NVDA

https://github.com/user-attachments/assets/a708fc96-c4b5-4813-8c35-3dacb5b472c7

JAWS

https://github.com/user-attachments/assets/9a8a122c-b3d7-4769-9628-4a429959a23c

**Comment by kapilvaishna** [comment]
@meganrogge In below environment, still showing the "Selected Model" and "Mode Multiplier" information for me. Could you please confirm how this issue fixed, information is removed or now it getting announce by screen reader.

VS code insider (1.105.0)

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/1419883d-2fa4-4bc7-8b3f-86bb25553cff" />

Not announced by  screen reader

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/55086864-d19a-430e-b91b-2598adca566b" />

**Comment by meganrogge** [comment]
You can now move focus to that via keyboard and the info is read 

https://github.com/user-attachments/assets/725b805d-7715-4369-b387-5b0be77cea14

**Comment by kapilvaishna** [comment]
@meganrogge I was checking with JAWS and NVDA but when focus move to that information it announces as 'note' for below version

VS code insider (1.105.0)

NVDA 2025.3

<img width="1365" height="712" alt="Image" src="https://github.com/user-attachments/assets/fb38b17b-29d7-4ce3-bea3-03eb609cecff" />


JAWS 2025

https://github.com/user-attachments/assets/2eef6123-121c-4272-84c9-afeb653d900a

**Comment by kapilvaishna** [comment]
Issue is not repro in VS code insider 1.106.0

https://github.com/user-attachments/assets/79d932cd-f293-4f0f-a6fd-aefb54807cd8
